### PR TITLE
feat: Test various handwritten fonts

### PR DESCRIPTION
**Pull request by lipis**
### Preview [fonts.excalidraw.com](https://fonts.excalidraw.com) - for #2945

-----

This is just an experiment to choose the font.. it won't be merged.

### CODE REVIEW

**Comment by j-f1** [comment]
Now if you change the font in the dorpdown, double-clicking the text element and clicking out of it will update the font!

**Review by arengoh — COMMENTED** [review]
Again

File: src/components/FontList.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  src: url("${path}");
+  font-display: swap;
+}
+`;
+
+const style = document.createElement("style");
+document.head.appendChild(style);
+
+export const FontList = ({
+  onChange,
```
Comment (arengoh) [inline src/components/FontList.tsx:24]: @j-f1 Can help me ?

**Review by arengoh — APPROVED** [review]

**Review by arengoh — COMMENTED** [review]
Can try again

**Comment by dwelle** [comment]
font now renders when you pick a font

File: src/testData.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      width: 499,
+      height: 175,
+      seed: 1845203046,
+      groupIds: [],
+      strokeSharpness: "sharp",
+      boundElementIds: ["7Slyz-S4KAlDN8LXnX_0h"],
+      fontSize: 20,
+      fontFamily: 1,
+      text:
+        "Latin\nPack my box with five dozen liquor jugs\nPACK MY BOX WITH FIVE DOZEN LIQUOR JUGS\npack my box with five dozen liquor jugs\n\n[ ] ( ) < > ? ! . , ; ' \" @ # $ € % ^ & * ~ ` / \\\n0 1 2 3 4 5 6 7 8 9",
```
Comment (Fausto95) [inline src/testData.js:31]:
```suggestion
        "Latin\nPack my box with five dozen liquor jugs\nPACK MY BOX WITH FIVE DOZEN LIQUOR JUGS\npack my box with five dozen liquor jugs\nVocê faz parte do meu coração.\nUn Cœur. Un maïs\n\n\n[ ] ( ) < > ? ! . , ; ' \" @ # $ € % ^ & * ~ ` / \\\n0 1 2 3 4 5 6 7 8 9",
```

**Pull request closed without merging**